### PR TITLE
adding SQS lambda handler code snippet in typescript and .net with batch item handling

### PR DESCRIPTION
**Pull request by tomeh100**
adding SQS lambda handler code snippet in typescript and .net with batch item handling

### CODE REVIEW

File: integration-sqs-to-lambda-with-batch-item-handling/snippet-data.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    {
+      "title": "Runtimes",
+      "codeTabs": [
+        {
+          "id": ".NET",
+          "title": "Usage Example with .NET:",
+          "description": "Consuming an SQS event with Lambda using .NET.",
+          "snippets": [
+            {
+              "snippetPath": "Function.cs",
```
Comment (boyney123) [inline integration-sqs-to-lambda-with-batch-item-handling/snippet-data.json:29]: Should this .NET example be here? I can't see the file commited?

Comment (tomeh100) [reply integration-sqs-to-lambda-with-batch-item-handling/snippet-data.json]: I updated snippet-data.json file to only include the typescript sample

**Comment by boyney123** [comment]
Think the branch has conflicts, can you take a look?

**Comment by tomeh100** [comment]
> done resolved the pending conflicts

**Comment by tomeh100** [comment]
> Think the branch has conflicts, can you take a look?

Resolved the conflict thanks

**Comment by jbesw** [comment]
Thanks - this will be live shortly.

**Pull request merged**